macos/ONLYOFFICE/update/updates/ONLYOFFICE/changes/8.1.0/changes.html

```diff
@@ -91,46 +91,83 @@ <h2>New Features</h2>
+        </ul>
+        <h4>PDF Editor</h4>
+        <ul>
+            <li>Added buttons for switching editing/viewing (annotations) modes to the toolbar and the editor header</li>
+            <li>Added tooltips for new or updated functionality (displayed when loading the editor or when switching to the corresponding tab)</li>
+            <li>In the Edit mode, it's possible to add various objects (using the <code>Home</code> and <code>Insert</code> tabs) and configure them using the right panel and context menu</li>
+            <li>Added the ability to add, delete or rotate pages using the context menu and the toolbar</li>
+            <li>Added a mini toolbar for adding annotations when selecting text</li>
         </ul>
 
         <h2>Fixes</h2>
 
-        <h4>All Editors</h4>
-        <ul>
-            <li>TODO</li>
-        </ul>
-        <h4>Document Editor</h4>
-        <ul>
-            <li>TODO</li>
-        </ul>
-        <h4>Spreadsheet Editor</h4>
-        <ul>
-            <li>TODO</li>
-        </ul>
-        <h4>Presentation Editor</h4>
-        <ul>
-            <li>TODO</li>
-        </ul>
-        <h4>Forms</h4>
+        <h4>Security</h4>
         <ul>
-            <li>TODO</li>
+            <li>Fixed the vulnerability in the fs.folderPath field which allows accessing the file system</li>
+            <li>Fixed vulnerabilities in the PtgName::assemble, PtgNameX::assemble, PtgParen::assemble, PtgRef3d::assemble, PtgList::assemble and PtgArea3d::assemble methods which cause crash when converting <code>XLS</code> to <code>XLSX</code></li>
+            <li>Fixed the vulnerability in the <code>CDataStream::ReadEmrTextBase</code> method which causes crash when converting <code>ODP</code> to <code>PDF</code></li>
+            <li>Fixed the vulnerability in the <code>GlobalsSubstream::UpdateDefineNames</code> method which causes crash when converting <code>XLS</code> to <code>XLSX</code></li>
+            <li>Fixed the vulnerability in the <code>WorkBookStream::UpdateXti</code>, <code>WorkBookStream::UpdateXti</code>, methods which causes crash when converting <code>XLS</code> to <code>XLSX</code></li>
+            <li>Fixed Heap Buffer Overflow when converting <code>EPUB</code> to <code>PDF</code></li>
+            <li>Fixed the vulnerability in the <code>CPPTUserInfo::LoadExternal</code> method which allows writing a file to a folder with restricted access when converting <code>PPT</code> to <code>PPTX</code></li>
+            <li>Fixed vulnerabilities which allow reading data from a third-party file when converting <code>OOXML</code> to <code>ODF</code> and vice versa</li>
+            <li>Fixed Heap Buffer Overflow in the <code>CSvmFile::Read_META_BMP</code> method when converting <code>ODP</code> to <code>PPTX</code></li>
+            <li>Fixed the vulnerability in the commandSetPassword method which allows setting a password for a document regardless of rights</li>
```

Comment:
Can not be reproduced in DesktopEditors.

Reply:
removed